--- phpids-0.5.4/lib/IDS/Config/Config.ini~	2008-12-18 16:46:32.000000000 +0200
+++ phpids-0.5.4/lib/IDS/Config/Config.ini	2009-04-15 01:43:33.443883790 +0300
@@ -9,17 +9,17 @@
     ; basic settings - customize to make the PHPIDS work at all
     filter_type     = xml
     
-    base_path       = /full/path/to/IDS/ 
+    base_path       = /usr/share/php/IDS
     use_base_path   = false
     
-    filter_path     = default_filter.xml
-    tmp_path        = tmp
+    filter_path     = /usr/share/php/IDS/default_filter.xml
+    tmp_path        = /tmp
     scan_keys       = false
 	
 	; in case you want to use a different HTMLPurifier source, specify it here
 	; By default, those files are used that are being shipped with PHPIDS
-	HTML_Purifier_Path	= IDS/vendors/htmlpurifier/HTMLPurifier.auto.php
-	HTML_Purifier_Cache = IDS/vendors/htmlpurifier/HTMLPurifier/DefinitionCache/Serializer
+	HTML_Purifier_Path	= /usr/share/php/HTMLPurifier.auto.php
+	HTML_Purifier_Cache = /usr/share/php/HTMLPurifier/DefinitionCache/Serializer
     
     ; define which fields contain html and need preparation before 
     ; hitting the PHPIDS rules (new in PHPIDS 0.5)
@@ -42,15 +42,15 @@
 [Logging]
 
     ; file logging
-    path            = tmp/phpids_log.txt
+    path            = /tmp/phpids_log.txt
 
     ; email logging
 
     ; note that enabling safemode you can prevent spam attempts,
     ; see documentation
-    recipients[]    = test@test.com.invalid
+    recipients[]    = root@localhost
     subject         = "PHPIDS detected an intrusion attempt!"
-    header			= "From: <PHPIDS> info@php-ids.org"
+    header			= "From: <PHPIDS> info@localhost"
     envelope        = ""
     safemode        = true
     urlencode       = true
@@ -59,8 +59,8 @@
     ; database logging
 
     wrapper         = "mysql:host=localhost;port=3306;dbname=phpids"
-    user            = phpids_user
-    password        = 123456
+    user            = mysql
+    password        = 
     table           = intrusions
 
 ; If you would like to use other methods than file caching you can configure them here
@@ -72,18 +72,18 @@
     expiration_time = 600
 
     ; file cache    
-    path            = tmp/default_filter.cache
+    path            = /tmp/default_filter.cache
 
     ; database cache
     wrapper         = "mysql:host=localhost;port=3306;dbname=phpids"   
-    user            = phpids_user
-    password        = 123456
+    user            = mysql
+    password        = 
     table           = cache  
 
     ; memcached     
     ;host           = localhost
     ;port           = 11211
     ;key_prefix     = PHPIDS
-    ;tmp_path       = tmp/memcache.timestamp
+    ;tmp_path       = /tmp/memcache.timestamp
 
 
